changes/2099.removal.rst

```diff
@@ -0,0 +1 @@
+The ``add_background_task()`` API on ``toga.App`` has been deprecated. Background tasks can be implemented using the new ``on_running`` event handler, or by using :any:`asyncio.create_task` or :any:`asyncio.loop.call_soon_threadsafe`.
```

Comment:
None of the `loop.call_...` methods create background tasks, because they only accept synchronous callables.

```suggestion
The ``add_background_task()`` API on ``toga.App`` has been deprecated. Background tasks can be implemented using the new ``on_running`` event handler, or by using :any:`asyncio.create_task`.
```

Reply:
Fair; the old call would also handle synchronous methods, so I was trying to be inclusive of that; but I guess that's a distraction.